File: Beecrowd/1238 Combinador/main.py
n = int(input())
for i in range(n):
    entrada = [str(a) for a in input().split()]
    s = entrada[0]
    t = entrada[1]

    new_string = ""
    
    tamS = len(s)
    tamT = len(t)

    # tam = tamS if tamS >= tamT else tamT

    if tamS >= tamT:
        tam = tamS
    else:
        tam = tamT

    for j in range(tam):
        if j < tamS:
            new_string += s[j]
        if j < tamT:
            new_string += t[j]
    
    print(new_string)

File: Beecrowd/1238 Combinador/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n, tam, tamS, tamT;
    string s, t, new_string;

    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> s >> t;
        tamS = s.size();
        tamT = t.size();

        //tam = s.size() >= t.size() ? tamS : tamT;

        if(s.size() >= t.size())
            tam = tamS;
        else
            tam = tamT;

        for(int i = 0; i < tam; i++){
            if(i < tamS){
                new_string += s[i];
            }
            if(i < tamT){
                new_string += t[i];
            }
        }
        cout << new_string << endl;
        new_string = "";
    }

    return 0;
}